%YAML:1.0
xarm_lite6:
  # 機器人採樣軌跡，笛卡爾座標 [x,y,z,rx,ry,rz]，旋轉角度單位為弧度
  sampleTrajectory:
      - [350, 0, 350, 3.139847324337799, 0.0, 0.0]
      - [279.7, 241.6, 287.3, 2.4329889772800954, -0.16929693744344995, 0.19722220547535924]
      - [328.3, 153.5, 325.9, 2.7663468644110125, -0.10821041362364843, 0.08028514559173915]
      - [338.5, 87.5, 360.1, 2.871066619530672, -0.029670597283903602, 0.08203047484373349]
      - [349.8, -23.6, 357.7, -3.0333822399661448, 0.0, 0.06283185307179587]
      - [328.1, -133.7, 313.9, -2.916445180082525, -0.15009831567151233, 0.17453292519943295]
      - [320.9, -180, 269.1, -2.8152160834668534, -0.17976891295541597, 0.16755160819145562]
      - [311.4, -210.7, 245.2, -2.7768188399229783, -0.20071286397934787, 0.12217304763960307]
      - [320.2, -220, 225, -2.6773350725593015, -0.20245819323134223, 0.12566370614359174]
      - [259.4, -56.7, 256, -3.0124382889422128, -0.35255650890285456, 0.08552113334772216]
      - [246, -17.6, 225.7, -3.085742117525975, -0.41015237421866746, 0.027925268031909273]
      - [180.6, -14.7, 240.1, 3.106686068549907, -0.5218534463463045, 0.04014257279586957]
      - [361.7, -34.3, 324.1, -2.9949849964222697, -0.0017453292519943296, 0.12391837689159739]
      - [190.2, -16.9, 233.6, 2.9949849964222697, -0.5515240436302081, 0.16755160819145562]
      - [289.6, -102.7, 280.7, -2.9740410453983377, -0.23387411976724018, 0.10122909661567112]
      - [164.1, -4.5, 269.8, -3.1276300195738385, -0.5270894341022875, 0.005235987755982988]
      - [271.9, -111.2, 253.7, -2.8675759610266836, -0.2775073510670984, 0.029670597283903602]
      - [285.3, -180.4, 226.2, -2.6598817800393584, -0.291469985083053, 0.10297442586766545]
      - [281.5, -192.9, 223.7, -2.607521902479528, -0.34033920413889424, 0.12915436464758037]
      - [243.1, 222.4, 248.7, 2.4085543677521746, -0.2984513020910304, 0.2548180707911721]
      - [278.2, 176.9, 278.1, 2.584832622203602, -0.2234021442552742, 0.2111848394913139]
      - [296.6, 94.4, 300.4, 2.9286624848464853, -0.21467549799530256, 0.1117010721276371]
      - [307.2, 20.4, 297.5, 3.0822514590219856, -0.17453292519943295, 0.04014257279586957]
      - [306.2, -27.1, 297.1, -3.1381019950858047, -0.16231562043547265, 0.03490658503988659]
      - [302.6, -55.6, 312.6, -3.005456971934235, -0.1605702911834783, 0.09773843811168245]
      - [350, 0, 350, 3.139847324337799, 0.0, 0.0]
      - [279.7, 241.6, 287.3, 2.4329889772800954, -0.16929693744344995, 0.19722220547535924]
      - [328.3, 153.5, 325.9, 2.7663468644110125, -0.10821041362364843, 0.08028514559173915]
      - [338.5, 87.5, 360.1, 2.871066619530672, -0.029670597283903602, 0.08203047484373349]
      - [349.8, -23.6, 357.7, -3.0333822399661448, 0.0, 0.06283185307179587]
      - [328.1, -133.7, 313.9, -2.916445180082525, -0.15009831567151233, 0.17453292519943295]
      - [320.9, -180, 269.1, -2.8152160834668534, -0.17976891295541597, 0.16755160819145562]
      - [311.4, -210.7, 245.2, -2.7768188399229783, -0.20071286397934787, 0.12217304763960307]
      - [320.2, -220, 225, -2.6773350725593015, -0.20245819323134223, 0.12566370614359174]
      - [259.4, -56.7, 256, -3.0124382889422128, -0.35255650890285456, 0.08552113334772216]
      - [246, -17.6, 225.7, -3.085742117525975, -0.41015237421866746, 0.027925268031909273]
      - [180.6, -14.7, 240.1, 3.106686068549907, -0.5218534463463045, 0.04014257279586957]
      - [361.7, -34.3, 324.1, -2.9949849964222697, -0.0017453292519943296, 0.12391837689159739]
      - [190.2, -16.9, 233.6, 2.9949849964222697, -0.5515240436302081, 0.16755160819145562]
      - [289.6, -102.7, 280.7, -2.9740410453983377, -0.23387411976724018, 0.10122909661567112]
      - [164.1, -4.5, 269.8, -3.1276300195738385, -0.5270894341022875, 0.005235987755982988]
      - [271.9, -111.2, 253.7, -2.8675759610266836, -0.2775073510670984, 0.029670597283903602]
      - [285.3, -180.4, 226.2, -2.6598817800393584, -0.291469985083053, 0.10297442586766545]
      - [281.5, -192.9, 223.7, -2.607521902479528, -0.34033920413889424, 0.12915436464758037]
      - [243.1, 222.4, 248.7, 2.4085543677521746, -0.2984513020910304, 0.2548180707911721]
      - [278.2, 176.9, 278.1, 2.584832622203602, -0.2234021442552742, 0.2111848394913139]
      - [296.6, 94.4, 300.4, 2.9286624848464853, -0.21467549799530256, 0.1117010721276371]
      - [307.2, 20.4, 297.5, 3.0822514590219856, -0.17453292519943295, 0.04014257279586957]
      - [306.2, -27.1, 297.1, -3.1381019950858047, -0.16231562043547265, 0.03490658503988659]
      - [302.6, -55.6, 312.6, -3.005456971934235, -0.1605702911834783, 0.09773843811168245]
    # - [352.3, -4.1, 318.8, 3.03, -0.38, 0.08]
    # - [314, -29.8, 215.4, 2.91, -0.63, 0.24]
    # - [352.3, -4.1, 318.8, 3.03, -0.38, 0.08]
    # - [314, -29.8, 215.4, 2.91, -0.63, 0.24]
    # - [352.3, -4.1, 318.8, 3.03, -0.38, 0.08]
    # - [314, -29.8, 215.4, 2.91, -0.63, 0.24]
    # - [352.3, -4.1, 318.8, 3.03, -0.38, 0.08]
    # - [314, -29.8, 215.4, 2.91, -0.63, 0.24]

  # 棋盤格資訊
  chessboard_info:
    pattern_width: 11  # 棋盤格內角點數量
    pattern_height: 8
    square_size: 20     # 棋盤格方格大小（單位：mm）
